edgecontext.thrift

```diff
@@ -104,6 +114,20 @@ struct RequestId {
     1: string readable_id
 }
 
+/** Locale data from a request to our services that we want to
+propagate between services.
+
+This model is a component of the "Edge-Request" header.  You should not need to
+interact with this model directly, but rather through the EdgeRequestContext
+interface provided by baseplate.
+
+*/
+struct Locale {
+    /** IETF language code representing the client locale preferences.
```

Comment:
can you add an example of `en_US` here? I see people occasionally use dash over underscore (e.g. `en-US`) and would like to explicitly call that out.

Reply:
Added in https://github.com/reddit/edgecontext/pull/12/commits/69bbb790bdce0842973f8af396c691e5568cc1df